### DISCUSSION TITLE
Publish assembly uber jar

### DISCUSSION THREAD

**Discussion by lilyevsky** [com-lihaoyi/mill#1267] [Q&A] [answered]
Using publishM2Local command from PublishModule. It publishes javadoc, sources, jar and pom files.
How do I publish the assembly uber jar produced by the "assembly" task?

**Comment by lefou** [comment, marked as answer]
You need to override the `extraPublish` target (from `PublishModule`) and include the assembly artifact with sensible metadata.

**Reply by lilyevsky** [reply]
Thanks @lefou , it worked.
